.circleci/deployment/commands.yml

```diff
@@ -71,28 +95,18 @@
               <<parameters.cf-space>>
 
   deploy-clamav:
-    parameters:
-      backend-appname:
-        default: tdp-backend
-        type: string
-      cf-org:
-        default: CF_ORG
-        type: env_var_name
-      cf-space:
-        default: tanf-dev
-        type: string
     steps:
       - run:
           name: Deploy ClamAV REST application
           command: |
             cf push clamav-rest -f tdrs-backend/manifest.clamav.yml \
-              --var cf-space=<<parameters.cf-space>> \
+              --var cf-space=tanf-prod \
       - run:
           name: Enable internal route between backend and clamav-rest app
           command: |
-            cf add-network-policy <<parameters.backend-appname>> clamav-rest \
-              -s <<parameters.cf-space>> \
-              -o ${<<parameters.cf-org>>} \
+            cf add-network-policy tdp-backend-prod clamav-rest \
+              -s tanf-prod \
+              -o hhs-acf-ofa \
```

Comment:
How do you add network-policy for each instance in dev/staging to connect to nginx router? We need something like:
cf add-network-policy "$CGAPPNAME_BACKEND" tdp-clamav-nginx-$env --protocol tcp --port 9000

Reply:
we are handling the network policies in deploy-backend.sh now